packages/.eslintrc.json

```diff
@@ -23,6 +23,7 @@
         "lodash/import-scope": ["error", "member"],
         "unused-imports/no-unused-imports-ts": "warn",
         "@typescript-eslint/await-thenable": "error",
-        "@typescript-eslint/no-for-in-array": "error"
+        "@typescript-eslint/no-for-in-array": "error",
+        "@typescript-eslint/consistent-indexed-object-style": ["error", "record"]
```

Comment:
Should we really prefer this one? It seems like we have many cases that the indexed key `{ [chainID: string]: T }` has its meaning of documenting/commenting the code meanwhile the `Record<string, T>` cannot represent.

Reply:
may need `@masknet/types`, provide shared type support system

Reply:
I mean, in the only case of `{ [key: string]: T }`, that `key` is a meaningless name, the choice `Record<string, T>` is good. In other cases, those names are used as the comment to the type.

Reply:
Agree with @Jack-Works. Both styles are acceptable.